doc/api/process.md

```diff
@@ -1819,6 +1819,46 @@ process.report.writeReport();
 
 Additional documentation is available in the [report documentation][].
 
+## process.resourcesUsage()
+<!-- YAML
+added: v12.x.x
+-->
+
+* Returns: {Object}
+    * `minflt` {integer}
+    * `majflt` {integer}
+    * `inblock` {integer}
+    * `oublock` {integer}
+    * `nvcsw` {integer}
+    * `nivcsw` {integer}
+
+The `process.resourcesUsage()` method return the current resources usage
+for the current process.
+To understand what these value mean, please refer to
+[getrusage(2)](http://man7.org/linux/man-pages/man2/getrusage.2.html)
+On Windows platform, only `inblock` and `oublock` are supported,
+all other fields will report zero as value.
+
+For example, the code:
+
+```js
+console.log(process.resourcesUsage());
+```
+
+Will generate:
+
+<!-- eslint-skip -->
+```js
+{
```

Comment:
This can be combined with the `console.log()` example, with the output commented out.